config/initializers/sidekiq.rb

```diff
@@ -21,3 +21,10 @@
     }
   end
 end
+
+# Sidekiq Cron
+schedule_file = "config/sidekiq_cron_schedule.yml"
+
+if File.exist?(schedule_file) && Sidekiq.server?
+  Sidekiq::Cron::Job.load_from_hash(YAML.load_file(schedule_file))
+end
```

Comment:
Should this be in the `if` block above?

Reply:
I'n not sure. We always want this enabled to update the trainees so wondering why it needs to have a toggle?

Reply:
moved this to settings https://github.com/DFE-Digital/register-trainee-teachers/pull/459/commits/2d862fef5f22796e0d2c367ac7bdd032d515bd9f